src/main/java/org/fundacion/jala/converter/core/facade/MetadataFacade.java

```diff
@@ -13,6 +13,8 @@
 
 import org.fundacion.jala.converter.core.ExtractMetadata;
 import org.fundacion.jala.converter.core.FileStorageService;
+import org.fundacion.jala.converter.core.exceptions.MetadataException;
+
```

Comment:
The empty line between the imports should be removed.

Reply:
Thanks, I have just fixed it